### ISSUE TITLE
[LUA] rttr:GetVersion() does not exist

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#1462]
The documentation references rttr:GetVersion() two times, but this function seems no longer to exist

https://github.com/Return-To-The-Roots/s25client/blob/master/doc/lua/methods.md#base-functions-callable-in-settings-and-game-mode

**Comment by Flamefire** [comment]
Hm, it should I'd say.

**Comment by Spikeone** [comment]
I get

```
Lua error: [string "function getRequiredLuaVersion()..."]:20: attempt to call method 'GetVersion' (a nil value)
```

When using this simple script:

```
function getRequiredLuaVersion()
    return 1.3
end

function onSettingsReady()
    rttr:Log("Version" ..rttr:GetVersion())
end
```

**Comment by Flamefire** [comment]
No I meant we should have it. But I think we replaced it by `GetFeatureLevel`.
The idea is: `getRequiredLuaVersion` should return an integer(!) which is then checked against the major version: https://github.com/Return-To-The-Roots/s25client/blob/4ce6199a77ebb44b5b14cdc23c0e0a19e41c77ca/libs/s25main/lua/LuaInterfaceGameBase.cpp#L44 as a change there means incompatible changes
`GetFeatureLevel` can then be used to check if a function or similar is available already.

Do you get what I mean? Do you mind updating the docs?

**Comment by Spikeone** [comment]
Yeah, I get the idea - can do, but you know that means sooner or later I will have a problem with cmake and ask you :D
